s/elements/stick/element.ts

```diff
@@ -57,14 +60,20 @@ export class NubStick extends MagicElement {
 		use.setup(setupWindowEvents({...starters, ...controls}))
 
 		return html`
+		<div
+			class="container"
+			@mousedown=${baseEvents.mousedown}
+			@touchstart=${baseEvents.touchstart}
+			@touchmove=${baseEvents.touchmove}
+			@touchend=${baseEvents.touchend}
+		>
```

Comment:
i think we could probably avoid this extra `.container` div -- can we move its functionality onto either the `nub-stick`/`:host` element, or otherwise the `.base` div?